wom/src/main/scala/wom/package.scala

```diff
@@ -1,7 +1,44 @@
+package wom
 
-import wdl.values.WdlValue
+import org.apache.commons.codec.digest.DigestUtils
 import wom.callable.Callable.InputDefinition
+import wom.values.WdlValue
+
+import scala.util.Try
+
+trait TsvSerializable {
+  def tsvSerialize: Try[String]
+}
+
+class WdlExpressionException(message: String = null, cause: Throwable = null) extends RuntimeException(message, cause)
+final case class OptionalNotSuppliedException(operationName: String) extends Exception(s"Sorry! Operation $operationName is not supported on empty optional values. You might resolve this using select_first([optional, default]) to guarantee that you have a filled value.")
+
+case class JobOutput(wdlValue: WdlValue)
+
+package object values {
+
+  type FileHasher = WdlFile => SymbolHash
 
-package object wom {
   type WomEvaluatedCallInputs = Map[InputDefinition, WdlValue]
+
+  implicit class HashableString(val value: String) extends AnyVal with Hashable {
+    def md5Sum: String = DigestUtils.md5Hex(value)
+    def md5SumShort: String = value.md5Sum.substring(0, 8)
```

Comment:
because sometime even md5 is just a bit too collision resistant?

Reply:
:man_shrugging: I just copy/pasted it, I can't vouch for it... 😛